counting-bits/hyejjun.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param {number} n
+ * @return {number[]}
+ */
+var countBits = function (n) {
+
+  let result = [];
+
+  for (let i = 0; i <= n; i++) {
+
+    let binary = i.toString(2);
```

Comment:
```suggestion
  const result = [];

  for (let i = 0; i <= n; i++) {

    const binary = i.toString(2);
```

재할당 되지 않을 변수는 const 키워드로 선언하는 편인데요. 이는 코드를 읽는 사람으로 하여금 이 변수는 재할당되지 않을 것임을 예측할 수 있도록 해주고, 나중에 실수로 해당 변수에 다른 값을 할당하게 되는 버그를 사전에 예방할 수 있기 때문이에요.